### PR TITLE
Feature/170642143/redirect to kim with filters

### PR DESCRIPTION
**Pull request by i-kushlianski**
- Fix `build:watch` script: it did not update Desktop on changes in hiro-ui (kudos to @yury-klika)
- Show buttons leading to KIM in KI widget
- Dispatch actions opening KIM with correct filter values
- Move React and React-dom to prod dependencies

### CODE REVIEW

File: packages/bootstrap/scripts/build.sh [hunk 1]
```diff
@@ -3,9 +3,9 @@
 # Build TS
 echo "-- Building Typescript"
 rm -rf ./dist
+rm -rf ./tsDist
```
Comment (yury-klika) [inline packages/bootstrap/scripts/build.sh]: Good catch

File: packages/bootstrap/src/components/dashboard/widgets/YourKis/KiRow.tsx [hunk 2]
```diff
@@ -29,15 +30,15 @@ export const KiRow: React.FC<KiRowProps> = ({
           <KisTitleText>{title}</KisTitleText>
         </KisTitle>
         <td>
-          {primary && (
+          {
```
Comment (yury-klika) [inline packages/bootstrap/src/components/dashboard/widgets/YourKis/KiRow.tsx]: No need for braces here

File: packages/bootstrap/src/components/dashboard/widgets/YourKis/KiRowDescription.tsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import React, { SyntheticEvent, useState } from 'react';
-
```
Comment (yury-klika) [inline packages/bootstrap/src/components/dashboard/widgets/YourKis/KiRowDescription.tsx]: This blank line should be restored - these imports are from different groups

Comment (i-kushlianski) [reply packages/bootstrap/src/components/dashboard/widgets/YourKis/KiRowDescription.tsx]: This was done automatially during `eslint --fix` I guess. Eslint actually could't correctly resolve our imports and keps complaining so I configured eslint and it fixed the issue. Pls re-check

**Pull request merged**